CHANGELOG.rst

```diff
@@ -6,6 +6,10 @@ v0.9.3
 ------
 
 - Add support for recursive requirements.
+- Add python 3.11 as a valid python version in choices.
+- Operating system and python version are now required fields in CLI.
+- Add dot versions (3.6, 3.7, 3.8, 3.9, 3.10, 3.11, 2.7) with 
```

Comment:
I am not sure I agree with this change. 
I get that using versions without dot in the CLI is not a great UX, but supporting both is likely a source of confusion and inconsitencies.

Reply:
https://github.com/nexB/python-inspector/commit/3a66ca2e30d8fd6e668219317723843f138145ab Removed this stuff which were not part of this PR